From 0b7c0c142a8c979b572bbdfdb3585c62a2922dd1 Mon Sep 17 00:00:00 2001
From: Frank Earl <frank@earlconsult.com>
Date: Fri, 4 May 2018 08:12:44 -0400
Subject: Turn on BT UART...

Stock, the config doesn't turn it on- so we're going to do that rather
than making the userland set everything up (Annoying...)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-h5-nanopi-neo-plus2.dts b/arch/arm64/boot/dts/allwinner/sun50i-h5-nanopi-neo-plus2.dts
index 6081035..691ea92 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-h5-nanopi-neo-plus2.dts
+++ b/arch/arm64/boot/dts/allwinner/sun50i-h5-nanopi-neo-plus2.dts
@@ -217,6 +217,12 @@
 	status = "okay";
 };
 
+&uart3 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&uart3_pins>, <&uart3_rts_cts_pins>;
+	status = "okay";
+};
+
 
 &r_pio {
 	wifi_en_npi: wifi_en_pin {
